README.md

```diff
@@ -1,2 +1,81 @@
 # nebula-flink-connector
 Flink Connector for Nebula Graph
+
+
+![](https://img.shields.io/badge/language-java-orange.svg)
+[![GitHub stars](https://img.shields.io/github/stars/vesoft-inc/nebula-flink-connector.svg?color=brightgreen)](https://GitHub.com/vesoft-inc/nebula-flink-connector/stargazers/)
+[![GitHub fork](https://img.shields.io/github/forks/vesoft-inc/nebula-flink-connector.svg?color=brightgreen)](https://GitHub.com/vesoft-inc/nebula-flink-connector/forks/)
+
+Nebula-Flink-Connector 2.0 is a connector that helps flink users to easily access Nebula Graph 2.0. If you want to access Nebula Graph 1.x with Flink, please refer to [Nebula-Flink-Connector 1.0](https://github.com/vesoft-inc/nebula-java/tree/v1.0/tools/nebula-flink) .
```

Comment:
helps Flink users

[Nebula-Flink-Connector 1.0](https://github.com/vesoft-inc/nebula-java/tree/v1.0/tools/nebula-flink). [Remove the extra space)